types/react-leaflet/index.d.ts

```diff
@@ -224,7 +224,7 @@ export interface MarkerProps extends MarkerEvents, Leaflet.MarkerOptions {
     children?: Children;
     position: Leaflet.LatLngExpression;
 }
-export class Marker<P extends MarkerProps, E extends Leaflet.Marker> extends MapLayer<P, E> {
+export class Marker<P = MarkerProps, E = Leaflet.Marker> extends MapLayer<P, E> {
```

Comment:
Since `MarkerProps` has a required property, `position`, type parameter `P` here should extend `MarkerProps`, as per our previous discussion.